commerce-cloud-code/app_yourshophere/cartridge/templates/resources/translations_de.properties

```diff
@@ -131,6 +131,9 @@ error.message.username.invalid=Ungültige E-Mail-Adresse.
 error.message.unable.to.create.account=Konto konnte nicht erstellt werden. Bitte versuchen Sie es erneut.
 error.oauth.login.failure=Anmeldung mit Social Media-Konto fehlgeschlagen.
 
+# Basket error messages
+error.basket.invalid=Ihr Warenkorb enthält Produkte ohne Preise. Bitte entfernen Sie diese Artikel, um mit dem Checkout fortzufahren.
```

Comment:
Wir haben die Kontrolle über unsere Daten verloren

Maybe we should auto-clean them and add it to the wishlist